### DISCUSSION TITLE
Failed to verify as a student with accurate, complete and valid information

### DISCUSSION THREAD

**Discussion by ZQDesigned** [community/community#78485] [GitHub Education] [answered]
### Hi everyone,

I am having problems trying to use **accurate, complete, and valid information** to verify my student status. I submitted the qualified verification materials as required for **over 5 times**, but I couldn't carry out the authentication smoothly. **The prompt for refusing validation was not clear enough** for me to continue to identify the problem and fix it.
![image](https://github.com/community/community/assets/89234036/22ab527c-1a57-417d-bc48-91ee1d0605ea)

**Comment by ft10101** [comment]
Contact the support team for the student verification platform, seek clarification on refusal messages, and ensure submitted documents meet official criteria for validation.

**Reply by ZQDesigned** [reply]
I was unable to contact the official support team because it indicated to me that "manual reviews are no longer available", prevented me from creating work orders and required me to submit to the GitHub Education community  I have repeatedly determined meets the official requirements for my submission

**Reply by ft10101** [reply, marked as answer]
By actively participating in the GitHub Education community discussions, you increase the chances of getting assistance from community members who may have faced similar issues. If your problem remains unresolved, you may consider exploring alternative channels provided by GitHub Education or reaching out to GitHub Support through any available means.

**Comment by mauricio-repetto** [comment]
@ZQDesigned how did you contact them? I'm having similar issues and I can't get this to work.
